### PR TITLE
update

### PR DESCRIPTION
**Pull request by Yolanda18-star**
@langyuemeng @YiniXu9506 @lichunzhu PTAL

### CODE REVIEW

File: go-dumpling-makes-exporteddata-stable.md [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+title: Go Dumpling! 让导出数据更稳定
+author: ['李淳竹']
+date: 2021-01-04
+summary: 在本文中，我们将会介绍一些 Dumpling 的进阶使用方法，帮助大家更稳定高效地导出数据。
+tags: ['Dumpling']
+---
+
+>作    者：李淳竹（lichunzhu），TiDB 研发工程师。
+
+>Migrate s SIG Community，主要涵盖 TiDB 数据处理工具，包含 TiDB 数据备份/导入导出，TiDB 数据变更捕获，其他数据库数据迁移至 TiDB 等，
```
Comment (lichunzhu) [inline go-dumpling-makes-exporteddata-stable.md:11]:
```suggestion
>Migrate SIG Community，主要涵盖 TiDB 数据处理工具，包含 TiDB 数据备份/导入导出，TiDB 数据变更捕获，其他数据库数据迁移至 TiDB 等，
```

File: go-dumpling-makes-exporteddata-stable.md [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+
+Dumpling 向云盘服务器发送数据很容易因为网络波动导致传输失败。Dumpling 通过为传输 client 设置合理的参数来重试传输数据的 HTTP 请求。通常情况下 Dumpling 会对可重试的错误尝试发送数据 3 次，每次重试间会有逐步增加并引入抖动的等待间隔。
+
+2.在数据库连接中断时进行重试
+
+在导出数据时数据库连接可能会不可避免地受到波动而中断。这时 Dumpling 会通过重建数据库连接的方式来尽量保证导出过程继续进行下去。
+
+这也会引出一个问题，Dumpling 提供了[不同的一致性选项](https://docs.pingcap.com/zh/tidb/stable/dumpling-overview#%E8%B0%83%E6%95%B4-dumpling-%E7%9A%84%E6%95%B0%E6%8D%AE%E4%B8%80%E8%87%B4%E6%80%A7%E9%80%89%E9%A1%B9)，贸然重建数据库连接将可能破坏导出快照的一致性。因此，Dumpling 针对不同的一致性配置做了不同的处理：
+
+   - onsistency 为 snapshot 或 none：
```
Comment (lichunzhu) [inline go-dumpling-makes-exporteddata-stable.md:66]:
```suggestion
   - consistency 为 `snapshot` 或 `none`：
```

File: go-dumpling-makes-exporteddata-stable.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    
+   这两种情况中，Dumpling 并没有为数据库上锁，Dumpling 会直接重建数据库连接。
+
+   - consistency 为 lock 或 flush：
```
Comment (lichunzhu) [inline go-dumpling-makes-exporteddata-stable.md:70]:
```suggestion
   - consistency 为 `lock` 或 `flush`：
```

**Review by lichunzhu — COMMENTED** [review]
Rest LGTM

**Review by lichunzhu — APPROVED** [review]
LGTM

**Comment by langyuemeng** [comment]
LGTM

**Review by YiniXu9506 — APPROVED** [review]
LGTM

File: go-dumpling-makes-exporteddata-stable.md [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+# TYPE dumpling_dump_finished_rows counter
+dumpling_dump_finished_rows 1.419506e+06
+# HELP dumpling_dump_finished_size counter for dumpling finished file size
+# TYPE dumpling_dump_finished_size counter
+dumpling_dump_finished_size 2.92577519e+08
+```
+
+在导出过程中，Dumpling 所用端口的 /metrics 地址也可以实时查询到 dumpling 目前已导出的行数与数据大小。
+
+Dumpling 将会始终倾听社区声音并持续改进。如果大家有更多改进意见，也欢迎大家到 [Dumpling repo](http://ithub.com/pingcap/dumpling) 提出 issue 与贡献 PR。
```
Comment (amyangfei) [inline go-dumpling-makes-exporteddata-stable.md:50]: link is broken
s/ithub/github

**Pull request merged**